streams/src/main/java/org/apache/kafka/streams/kstream/NamedOperation.java

```diff
@@ -17,7 +17,7 @@
 package org.apache.kafka.streams.kstream;
 
 /**
- * Default interface which can be used to personalized the named of operations, internal topics or store.
+ * Default interface which can be used to customize the named of operations, internal topics or store.
```

Comment:
shall we also change the description to 'the name of operations'?

Reply:
Thanks @johnnychhsu, fixed.